docs/manual/en/introduction/Installation.html

```diff
@@ -109,21 +110,11 @@ <h2>Examples</h2>
 		</code>
 
 		<p>
-			If three.js was installed from a CDN, use the same CDN to install other components:
+			If three.js was installed from a CDN, use the same code, but with `three/addons/` in the import map.
 		</p>
 
-		<code>
```

Comment:
Please restore this code listing with an enhanced import map that shows how to configure `three/addons/` and how to import `OrbitControls`.

Reply:
Done.